install.md

```diff
@@ -17,7 +17,15 @@ Select the operating system you are working on.
 
 <div id="tab-osx" class="content active">
 
-[Node.js](http://nodejs.org/) is a prerequisite for installing the Tessel command line. You can install it from the [official website](http://nodejs.org/), or if you have [brew](http://brew.sh/) installed: `brew install node`
+[Node.js](http://nodejs.org/) is a prerequisite for installing the Tessel command line. You can install it using nvm by running the following commands in your terminal:
+
+`curl https://raw.githubusercontent.com/creationix/nvm/v0.23.2/install.sh | bash`
+
+`source ~/.bashrc`
+
+`nvm install 0.12.7`
```

Comment:
Yeah, I linked to nvm-windows in the Windows implementation - because it's really hard to find the different versions on Node's site. Is nvm-windows not a good solution?